tbtadm/controller.cpp

```diff
@@ -220,10 +223,9 @@ int findSL()
 {
     if (fs::exists(sysfsDevicesPath))
     {
-        for (auto dir : boost::make_iterator_range(
-                 fs::directory_iterator(sysfsDevicesPath), {}))
+        for (auto& dir : fs::directory_iterator(sysfsDevicesPath))
```

Comment:
I'm trying to understand why I used `make_iterator_range` in first place and can't remember...
Thanks!